### PR TITLE
feat(core): install litellm by default

### PR DESCRIPTION
**Pull request by mhordynski**

### CODE REVIEW

**Review by ludwiktrammer — APPROVED** [review]

File: packages/ragbits-core/src/ragbits/core/embeddings/litellm.py [hunk 1]
```diff
@@ -44,9 +39,6 @@ def __init__(
         Raises:
             ImportError: If the 'litellm' extra requirements are not installed.
         """
-        if not HAS_LITELLM:
-            raise ImportError("You need to install the 'litellm' extra requirements to use LiteLLM embeddings models")
```
Comment (micpst) [inline packages/ragbits-core/src/ragbits/core/embeddings/litellm.py:48]: update docstring pls, we no longer rise exception here

File: packages/ragbits-core/src/ragbits/core/llms/clients/litellm.py [hunk 2]
```diff
@@ -68,9 +61,6 @@ def __init__(
         Raises:
             ImportError: If the 'litellm' extra requirements are not installed.
         """
-        if not HAS_LITELLM:
-            raise ImportError("You need to install the 'litellm' extra requirements to use LiteLLM models")
```
Comment (micpst) [inline packages/ragbits-core/src/ragbits/core/llms/clients/litellm.py:72]: ditto

File: packages/ragbits-core/src/ragbits/core/llms/litellm.py [hunk 3]
```diff
@@ -51,9 +46,6 @@ def __init__(
         Raises:
             ImportError: If the 'litellm' extra requirements are not installed.
         """
-        if not HAS_LITELLM:
-            raise ImportError("You need to install the 'litellm' extra requirements to use LiteLLM models")
```
Comment (micpst) [inline packages/ragbits-core/src/ragbits/core/llms/litellm.py:55]: ditto

Comment (mhordynski) [reply packages/ragbits-core/src/ragbits/core/llms/litellm.py:55]: good catch, thx

**Review by micpst — APPROVED** [review]

**Pull request merged**